ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/RecursivePeerRefreshState.java

```diff
@@ -191,7 +191,10 @@ void onNeighboursReceived(final DiscoveryPeer peer, final List<DiscoveryPeer> pe
     if (metadataPeer == null) {
       return;
     }
-    LOG.debug("Received neighbours packet with {} neighbours", peers.size());
+    LOG.info(
```

Comment:
Could this be noisy? Any idea how often this would be printed in the normal case?

Reply:
changed back